server/aesdsocket-start-stop.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/sh
```

Comment:
@nihalthirunakarasu make sure to remove your aesdsocket executable from your repo and add to .gititnore.